docs/configuration-reference/platforms/tinkerbell.md

```diff
@@ -0,0 +1,185 @@
+  region         = var.state_s3_region
+  dynamodb_table = var.lock_dynamodb_table
+}
+
+# backend "local" {
+#   path = "path/to/local/file"
+#}
+
+variable "asset_dir" {}
+variable "cluster_name" {}
+variable "dns_zone" {}
+variable "ssh_public_keys" {}
+variable "controller_ip_addresses" {}
+variable "controller_clc_snippets" {}
+variable "controller_flatcar_install_base_url" {}
+variable "os_channel" {}
+variable "os_version" {}
+variable "hosts_cidr" {}
+variable "flatcar_image_path" {}
+variable "pool_path" {}
+variable "enable_aggregation" {}
+variable "enable_reporting" {}
+variable "pod_cidr" {}
+variable "service_cidr" {}
+variable "cluster_domain_suffix" {}
+variable "certs_validity_period_hours" {}
+variable "network_mtu" {}
+variable "disable_self_hosted_kubelet" {}
+variable "ip_addresses" {}
+variable "flatcar_install_base_url" {}
+variable "clc_snippets" {}
+variable "labels" {}
+variable "taints" {}
+
+cluster "tinkerbell" {
+  asset_dir = var.asset_dir
+
+  name = var.cluster_name
+
+  dns_zone = var.dns_zone
+
+  ssh_public_keys = var.ssh_public_keys
+
+  controller_ip_addresses = var.controller_ip_addresses
+
+  controller_clc_snippets = var.controller_clc_snippets
+
+  controller_flatcar_install_base_url = var.controller_flatcar_install_base_url
+
+  os_channel = var.os_channel
+
+  os_version = var.os_version
+
+  experimental_sandbox {
+    hosts_cidr         = var.hosts_cidr
+    flatcar_image_path = var.flatcar_image_path
+    pool_path          = var.pool_path
+  }
+
+  enable_aggregation = var.enable_aggregation
+
+  disable_self_hosted_kubelet = var.disable_self_hosted_kubelet
+
+  enable_reporting = var.enable_reporting
+
+  pod_cidr = var.pod_cidr
+
+  service_cidr = var.service_cidr
+
+  cluster_domain_suffix = var.cluster_domain_suffix
+
+  certs_validity_period_hours = var.certs_validity_period_hours
+
+  network_mtu = var.network_mtu
+
+  disable_self_hosted_kubelet = var.disable_self_hosted_kubelet
+
+  worker_pool "pool1" {
+    ip_addresses = var.ip_addresses
+
+    ssh_public_keys = var.ssh_public_keys
+
+    os_channel = var.os_channel
+
+    os_version = var.os_version
+
+    flatcar_install_base_url = var.flatcar_install_base_url
+
+    clc_snippets = var.clc_snippets
+
+    labels = var.labels
+
+    taints = var.taints
+  }
+}
+```
+
+## Attribute reference
+
+| Argument                                  | Description                                                                                                                                                                                                                                | Default         | Type         | Required |
+|-------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-:-:-------------|-:-:----------|-:-:------|
```

Comment:
nit, maybe table formatting off [here](https://github.com/kinvolk/lokomotive/blob/0f927098bf2d17a002f32f47852aa716afe3844a/docs/configuration-reference/platforms/tinkerbell.md#attribute-reference), perhaps should be: `|:-----:|`

Reply:
Thanks, it was copy-pasted from some other place and it breaks GitHub table rendering. Replaced with `|---|` only as we do for other tables.